vulture_os/services/rsyslogd/config/rsyslog_template_sentinel_ama.conf

```
@@ -0,0 +1,74 @@
+template(name="sentinel_ama" type="list" option.jsonf="on") {
+    constant(value="{")
+    constant(value="\"Message\":\"") property(name="msg")
+    constant(value="\",\"Host\":\"") property(name="hostname")
+    constant(value="\",\"Severity\":\"") property(name="syslogseverity-text")
+    constant(value="\",\"SeverityNumber\":\"") property(name="syslogseverity")
+    constant(value="\",\"Facility\":\"") property(name="syslogfacility-text")
+    constant(value="\",\"EventTime\":\"") property(name="timegenerated")
+    constant(value="\",\"Timestamp\":\"") property(name="timestamp")
+    constant(value="\",\"ProcessId\":\"") property(name="procid")
+    constant(value="\",\"ident\": \"{{ node.name }}\"")
+    constant(value=",\"HostIP\":\"") property(name="fromhost-ip")
+    constant(value="\"}")
+}
```

missing carriage returns
```suggestion
    constant(value="\"}\n")
}
```